introduction.md

```diff
@@ -0,0 +1,3 @@
+Dojo (道場 dōjō?) is a Japanese term which literally means "place of the way". Initially, dōjōs were adjunct to temples.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.